src/components/filter/filter.component.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './filter.styles.scss';
+import { connect } from 'react-redux';
+import { changeFilter } from '../../redux/filter/filter.actions';
+import { toggleHidden } from '../../redux/filter/filter.actions';
+import { CATEGORIES } from '../../constants/constants';
+
+const Filter = ({changeFilter, toggleHidden}) => (
+	<div className='filter-box' onClick={toggleHidden}>
+		<div className='categories'>
+			<p className='category' value="All" onClick ={() => changeFilter("") }>All</p>
+			{
+				CATEGORIES.map((category, i) => (
+					<p key={i} className='category' onClick ={() => changeFilter(category) }>{category}</p>
+				))
+			}
+		</div>
+	</div>
+)
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
Ditto, we can use a simplified way to map dispatch to our props by just including an object `{changeFilter, toggleHidden}` as a second argument to `connect`